Makefile

```
@@ -100,50 +100,50 @@ clean: clean-deps clean-dist clean-build
 
 ##
```

```suggestion
##---------------------
```
We need this to separate into multiple sections. Otherwise, it will be very hard to read.